openadapt/app/objects/local_file_picker.py

```diff
@@ -1,4 +1,14 @@
-# retrieved from https://github.com/zauberzeug/nicegui/tree/main/examples/local_file_picker
```

Comment:
Can we include the source URL in the docstring?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507